templates/jlink/README.md

```diff
@@ -18,26 +20,31 @@ prepare UBI9 OpenJDK ImageStreams with `jlink-dev` support.
 
         cekit --descriptor ubi9-openjdk-17.yaml build docker
 
-2. Within your OpenShift project,
+2. Create an OpenShift project and namespace
+
+        oc new-project jlink-dev
+        oc create namespace jlink
+
+3. Within your OpenShift project,
 
-        oc create imagestream ubi9-openjdk-17
+        oc create imagestream openjdk-17-jlink-tech-preview
 
-3. You may need to configure your container engine to not TLS-verify the OpenShift
+4. You may need to configure your container engine to not TLS-verify the OpenShift
    registry. For Docker, add the following to `/etc/docker/daemon.json` and restart
    the daemon:
 
         {
           "insecure-registries": [ "default-route-openshift-image-registry.apps-crc.testing" ]
         }
 
-4. Log into the OpenShift registry, e.g.
+5. Log into the OpenShift registry, e.g.
 
         REGISTRY_AUTH_PREFERENCE=docker oc registry login
 
-5. tag and push the dev image into it. The OpenShift console gives you the
+6. tag and push the dev image into it. The OpenShift console gives you the
    exact URI for your instance
 
-        docker tag ubi9/openjdk-17:1.18 default-route-openshift-image-registry.apps-crc.testing/jlink1/ubi9-openjdk-17:1.18
+        docker tag openjdk-tech-preview/openjdk-17-jlink-rhel9:1.18 default-route-openshift-image-registry.apps-crc.testing/jlink1/openjdk-17-jlink-tech-preview:1.18
```

Comment:
Update reference to be podman and openjdk-21
The image tag must include the project name from above. So if you create oc new-project jlink1 then this would be correct. If you created a project named jlink-dev then the image with this tag will not be allowed to push to OCP as you will get a denied error because there is no namespace of that name present.

Reply:
let's use a variable in the docs for the project name, starting with `oc new-project $PROJECT` or similar, and using `$PROJECT` elsewhere